nucypher/cli/painting.py

```diff
@@ -380,6 +382,57 @@ def paint_staking_confirmation(emitter, staker, new_stake):
     emitter.echo(next_steps, color='green')
 
 
+def paint_stakes(emitter, stakeholder, paint_inactive: bool = False, staker_address: str = None):
+    headers = ('Idx', 'Value', 'Remaining', 'Enactment')
+    staker_headers = ('Status', 'Restaking', 'Winding Down', 'Unclaimed Fees')
+
+    stakers = stakeholder.get_stakers()
+    if not stakers:
+        emitter.echo("No staking accounts found.")
+
+    total_stakers = 0
+    for staker in stakers:
+        if not staker.stakes:
+            # This staker has no active stakes.
+            # TODO: Something with non-staking accounts?
+            continue
+
+        # Filter Target
+        if staker_address and staker.checksum_address != staker_address:
+            continue
+
+        stakes = sorted(staker.stakes, key=lambda s: s.address_index_ordering_key)
+        active_stakes = filter(lambda s: s.is_active, stakes)
+        if not active_stakes:
+            emitter.echo(f"There are no active stakes\n")
+
+        fees = staker.policy_agent.get_reward_amount(staker.checksum_address)
+        gwei_fees = f"{Web3.fromWei(fees, 'gwei')} Gwei"
+        last_confirmed = staker.staking_agent.get_last_active_period(staker.checksum_address)
+        missing = staker.staking_agent.get_missing_confirmations(staker_address=staker.checksum_address)
+
+        staker_data = [f'Missing {missing} confirmation{"s" if missing > 1 else ""}' if missing else f'Confirmed #{last_confirmed}',
+                       f'{"Yes" if staker.is_restaking else "No"} ({"Locked" if staker.restaking_lock_enabled else "Unlocked"})',
```

Comment:
We talked about potentially changing the term "Locked" vs "Unlocked"...I think it was more along the lines of modifiable/configurable or something similar - https://github.com/nucypher/nucypher/issues/1618.